plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -78,9 +79,13 @@ const refCompare = (a: Entity, b: Entity) => {
 
 /** @public */
 export const CatalogTable = (props: CatalogTableProps) => {
-  const { columns, actions, tableOptions, subtitle, emptyContent } = props;
+  const { tableOptions, subtitle, emptyContent } = props;
   const { isStarredEntity, toggleStarredEntity } = useStarredEntities();
-  const { loading, error, entities, filters } = useEntityList();
+  const { loading, error, entities, filters, pageInfo } = useEntityList();
+
+  const enablePagination = !!pageInfo;
+
+  const showTypeColumn = filters.type === undefined;
```

Comment:
Closing over this is gonna mess with column customization, I think we need a different way to handle it